frontend/package.json

```diff
@@ -25,9 +25,11 @@
     "@babel/runtime": "^7.16.0",
     "@emotion/react": "^11.5.0",
     "@emotion/styled": "^11.3.0",
+    "@material-ui/icons": "^4.11.2",
     "@mui/icons-material": "^5.1.0",
     "@mui/material": "^5.0.6",
     "@mui/styles": "^5.1.0",
+    "clsx": "^1.1.1",
```

Comment:
I see that clsx is not used in the project. Unused dependencies should be removed.